src/demos/storage-access-api/personalization.ejs

```diff
@@ -9,90 +9,122 @@
     containerClass,
     hasStorageAccess;
 
+  // Fetch and apply the user's preferred theme from a remote URL
   async function fetchAndApplyTheme() {
     try {
+      // Send a GET request to the remote URL
       const response = await fetch(`${baseURL}/get-personalization`, {
         method: 'GET',
+        // Include the user's credentials in the request
         credentials: 'include',
     });
-
+      // Check if the response was successful
       if (!response.ok) {
         throw new Error('Network response was not ok');
+        // If the response was not successful, throw an error
```

Comment:
This comment should go one line above, comments are added before the line instead of after the line.